src/components/editor-page/editor-page-content.tsx

```diff
@@ -10,12 +10,9 @@ import { useApplyDarkMode } from '../../hooks/common/use-apply-dark-mode'
 import { updateNoteTitleByFirstHeading } from '../../redux/note-details/methods'
 import { MotdModal } from '../common/motd-modal/motd-modal'
 import { AppBar, AppBarMode } from './app-bar/app-bar'
-import { EditorMode } from './app-bar/editor-view-mode'
-import { useViewModeShortcuts } from './hooks/use-view-mode-shortcuts'
 import { Sidebar } from './sidebar/sidebar'
 import { Splitter } from './splitter/splitter'
 import type { DualScrollState, ScrollState } from './synced-scroll/scroll-props'
-import { useEditorModeFromUrl } from './hooks/use-editor-mode-from-url'
```

Comment:
If I recall correctly this feature is also necessary to send a specific link that sets the correct editor mode.